connectors/flink/src/main/java/io/delta/flink/internal/table/DeltaCatalogTableHelper.java

```diff
@@ -324,6 +289,41 @@ public static Map<String, String> prepareDeltaTableProperties(
         return deltaLogProperties;
     }
 
+    /**
+     * Prepare catalog table to store in metastore. This table will have only selected
+     * options from DDL and an empty schema.
+     */
+    public static ResolvedCatalogTable prepareMetastoreTable(
```

Comment:
Hey - why move this code from above to below, here?

Could you edit it in the same place in the file? This makes seeing the git diff much easier